tools/lib/sanity_check.py

```diff
@@ -3,11 +3,17 @@
 
 import sys
 
+import getpass
+
 def check_venv(filename):
     # type: (str) -> None
     try:
         import ujson
     except ImportError:
         print("You need to run %s inside a Zulip dev environment." % (filename,))
-        print("If you are using Vagrant, you can `vagrant ssh` to enter the Vagrant guest.")
+        username = getpass.getuser()
+        if username != 'vagrant':
+            print("If you are using Vagrant, you can `vagrant ssh` to enter the Vagrant guest.")
+        else:
+            print("You can `source /srv/zulip-venv/bin/activate` to enter dev environment.")
```

Comment:
I'd prefer:

"You can `source /srv/zulip-venv/bin/activate` to enter the dev environment."